random_menu_generator.rb

```diff
@@ -0,0 +1,47 @@
+# thanks to Kari - figured out RegEx
+# testing that the user inputted only words and not numbers/symbols/etc.
+def validate_alpha_chars(array)
+  array.each do |word|
```

Comment:
I inverted the regex matcher because it was matching any input that included a alpha character (e.g. "9483c" would pass) and turned it into a method so I could check every instance of user input as soon as it was entered.